seeders/20220422184411-create-demo-activities.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.bulkInsert('Activities', [{
+      name: 'Activitie test',
+      content: 'Demo for activities',
+      image: 'https://www.designevo.com/res/templates/thumb_small/colorful-hand-and-warm-community.png',
+      createdAt: new Date,
+      updatedAt: new Date
+    }], {});
+
+    await queryInterface.bulkInsert('Activities', [{
+      name: 'Activitie test 2',
+      content: 'Demo for activities 2',
+      image: 'https://www.designevo.com/res/templates/thumb_small/colorful-hand-and-warm-community.png',
+      createdAt: new Date,
+      updatedAt: new Date
+    }], {});
+
+  },
+
+  down: async (queryInterface, Sequelize) => {
+    /**
```

Comment:
Faltó el revert del seed.
https://sequelize.org/docs/v6/other-topics/migrations/#creating-the-first-seed

Reply:
Ya agregué el revert.